Documentation/manual/ModelAdjust.md

```
@@ -73,6 +73,19 @@ PLATEAUウィンドウの`モデル調整`では、インポートした都市
   - 選択肢から任意個のパッケージを選んで`決定`を押して選択できます。
 - 追加したオブジェクトは`除く`ボタンで対象から除きます。
 
+#### 動的タイルを対象とした選択
+
+動的タイルを分割／結合／マテリアル分けの対象として指定することができます。  
+動的タイルを扱う際には次のような操作が可能です。
+
+![](../resources/manual/modelAdjust/DynamicTiles.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)